extension/js/agent/utils/serializeObject.js

```diff
@@ -84,7 +83,7 @@
 
   Agent.serializeClass = function(object, info, shouldMemoize) {
 
-    var serializeObject = objectPath(object, info.path);
+    var serializeObject = this.objectPath(object, info.path);
 
     if (this.classPropertyCache[info.name]
```

Comment:
Good catch.  These were two where I had to use `this` on earlier instances because they weren't `Agent` wrapped yet and I must've kept going